app/scripts/services/dimBungieService.factory.js

```diff
@@ -143,24 +144,27 @@
 
     function processBnetPlatformsRequest(response) {
       if (response.data.ErrorCode === 99) {
-        chrome.tabs.query({
-          'url': '*://*.bungie.net/*'
-        }, function(tabs) {
-          if (_.size(tabs) === 0) {
-            chrome.tabs.create({
-              url: 'http://bungie.net',
-              active: false
-            });
-          }
-        });
-
+        try {
+          //TODO: This is listed as supported on the firefox developer docs but throws a malformed URI error
+          chrome.tabs.query({
+            'url': '*://*.bungie.net/'
+          }, function(tabs) {
```

Comment:
We actually can't do this from chrome anymore either - we don't ask for the permission. If you rebase it should be gone.